typescript/widgets/package.json

```diff
@@ -7,14 +7,26 @@
     "react-dom": "^18"
   },
   "dependencies": {
+    "@cosmos-kit/react": "^2.18.0",
     "@headlessui/react": "^2.1.8",
     "@hyperlane-xyz/sdk": "7.0.0",
     "@hyperlane-xyz/utils": "7.0.0",
+    "@interchain-ui/react": "^1.23.28",
+    "@rainbow-me/rainbowkit": "^2.2.0",
+    "@solana/wallet-adapter-react": "^0.15.32",
+    "@solana/wallet-adapter-react-ui": "^0.9.31",
+    "@solana/web3.js": "^1.95.4",
     "clsx": "^2.1.1",
-    "react-tooltip": "^5.28.0"
+    "react-tooltip": "^5.28.0",
+    "viem": "^2.21.41",
+    "wagmi": "^2.12.26"
   },
   "devDependencies": {
-    "@hyperlane-xyz/registry": "4.7.0",
+    "@chakra-ui/react": "^2.8.2",
+    "@cosmjs/cosmwasm-stargate": "^0.32.4",
+    "@emotion/react": "^11.13.3",
+    "@emotion/styled": "^11.13.0",
```

Comment:
Unfortunately we need all this to use and test CosmosKit but at least they're only dev deps